CONTRIBUTING.md

```diff
@@ -124,6 +124,8 @@ If your change address an issue listed in GitHub, please use `Fixes:` tag with t
 	Fixes: #339
 ```
 
+The `Fixes:` tags should be put in the end of the detailed description.
+
```

Comment:
I think it should be `at the end` and not `in the end`.

Reply:
Probably you are right, will re-push soon

Reply:
Maybe also clarify "detailed description" to "detailed commit description".

Reply:
Would too much IMHO. It's in the section about the commit descriptions and line after this sentence there is a pointer to an article about good commit messages.

Reply:
Ok, agreed.